picard/ui/options/general.py

```diff
@@ -62,12 +62,10 @@ class GeneralOptionsPage(OptionsPage):
         TextOption("setting", "server_host", MUSICBRAINZ_SERVERS[0]),
         IntOption("setting", "server_port", 443),
         BoolOption("setting", "use_server_for_submission", False),
-        TextOption("persist", "oauth_refresh_token", ""),
         BoolOption("setting", "analyze_new_files", False),
         BoolOption("setting", "ignore_file_mbids", False),
         TextOption("persist", "oauth_refresh_token", ""),
         TextOption("persist", "oauth_refresh_token_scopes", ""),
-        TextOption("persist", "oauth_access_token", ""),
```

Comment:
I don't see a second declaration of oauth_access_token, is it not needed?

Reply:
> I don't see a second declaration of oauth_access_token, is it not needed?

No idea why this was committed, I guess lack of coffee. Fixed, forced pushed.